src/sky/pro/average/Main.java

```diff
@@ -0,0 +1,128 @@
+        calculateTotalSum(employees, kindDepartment);
+        printTotalSum(calculateTotalSum(employees, kindDepartment));
+        calculateAverageSum(sum, employees, kindDepartment);
+        indexSumDepartment(employees, indexInterest, kindDepartment);
+        listMorePargSalary(employees, pargSalary);
+        listLessPargSalary(employees, pargSalary);
+    }
+
+    public static void indexSum(Employee[] employees, int indexInterest) {
+        for (int i = 0; i < employees.length; i++) {
+            double indexSum = (employees[i].getSalary()) + (employees[i].getSalary() * indexInterest / 100);
+            System.out.println(employees[i].getFullName() + " " + indexSum);
+        }
+    }
+    public static void printList(Employee[] employees, int kindDepartment) {
+        for (int i = 0; i < employees.length; i++) {
+            int id = countAdd();
+            if (employees[i].getDepartment() == kindDepartment) {
+                System.out.println(id + " " + " " + employees[i].getFullName() + " " + employees[i].getSalary());
+            }
+        }
+    }
+
+    public static void getEmployeeMaxSum(Employee[] employees, int kindDepartment) {
+        int maxSum = 0;
+        String luckyEmployee = "";
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i].getDepartment() == kindDepartment && employees[i].getSalary() > maxSum) {
+                luckyEmployee = employees[i].getFullName();
+                maxSum = employees[i].getSalary();
+            }
+        }
+        System.out.println("Сотрудник с максимальной зарплатой" + " " + maxSum + " " + luckyEmployee);
+    }
+
+    public static void getEmployeeMinSum(Employee[] employees, int kindDepartment) {
+        int minSum = employees[0].getSalary();
+        String unluckyEmployee = "";
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i].getDepartment() == kindDepartment && employees[i].getSalary() < minSum) {
+                minSum = employees[i].getSalary();
+                unluckyEmployee = employees[i].getFullName();
+            }
+        }
+        System.out.println("Сотрудник с минимальноой зарплатой" + " " + minSum + " " + unluckyEmployee);
+    }
+
+    static double calculateTotalSum(Employee[] employees, int kindDepartment) {
+        double sum = 0;
+        for (int i = 0; i < employees.length; i++) {
+            if (employees[i].getDepartment() == kindDepartment) {
+                sum += employees[i].getSalary();
+            }
+        }
+        return sum;
+    }
+
+    static void printTotalSum(double sum) {
+        System.out.println("Cумма затрат на зарплаты в месяц " + sum);
+    }
+
+    static void calculateAverageSum(double sum, Employee[] employees, int kindDepartment) {
+        int count = 0;
+        for (int i = 0; i < employees.length; i++) {
+            boolean isNeedDepartment = employees[i].getDepartment() == kindDepartment;
+            if (isNeedDepartment == true) {
```

Comment:
```suggestion
            if (isNeedDepartment) {
```
сравнение выдает нам boolean значение, мы можем ограничиться самой переменной типа boolean